plugins/pur/core/pom.xml

```diff
@@ -166,14 +164,12 @@
     <dependency>
       <groupId>org.slf4j</groupId>
       <artifactId>slf4j-api</artifactId>
-      <version>${slf4j-api.version}</version>
-      <scope>provided</scope>
     </dependency>
     <dependency>
       <groupId>org.slf4j</groupId>
       <artifactId>slf4j-log4j12</artifactId>
-      <version>${slf4j-log4j12.version}</version>
-      <scope>provided</scope>
+      <version>${slf4j.version}</version>
```

Comment:
The version is being managed in the parent pom. I'm not 100% sure but I think you don't need to declare it here just to change the scope.